queries.sql

```diff
@@ -2,16 +2,32 @@
 
 -- Find all customers with postal code 1010
 
+SELECT * FROM Customers WHERE PostalCode = 1010;
+
 -- Find the phone number for the supplier with the id 11
 
+SELECT Phone FROM Suppliers WHERE SupplierID = 11
+
 -- List first 10 orders placed, sorted descending by the order date
 
+SELECT * FROM Orders ORDER BY OrderDate DESC LIMIT 10
+
 -- Find all customers that live in London, Madrid, or Brazil
 
+SELECT * FROM Customers WHERE City IN ("London", "Madrid") OR Country = "Brazil"
```

Comment:
I think you're the first that's discovered you can use parenthesis for more than one variable. Good job!